### PR TITLE
Code refactor: replace repeated values with a constant

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Replace repeated values with a constant

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/upgrade/prepull_test.go [hunk 1]
```diff
@@ -26,6 +26,10 @@ import (
 	//"k8s.io/apimachinery/pkg/util/version"
 )
 
+const (
```
Comment (yagonobre) [inline cmd/kubeadm/app/phases/upgrade/prepull_test.go]: Please move it to `cmd/kubeadm/app/constants/constants.go`

**Review by yagonobre — COMMENTED** [review]
Thanks @SataQiu 
/priority backlog

File: cmd/kubeadm/app/phases/upgrade/prepull_test.go (hunk 1 above)
Comment (SataQiu) [reply cmd/kubeadm/app/phases/upgrade/prepull_test.go]: @yagonobre Thanks for your feedback. All have been done.

File: cmd/kubeadm/app/phases/upgrade/prepull_test.go [hunk 2]
```diff
@@ -116,25 +116,25 @@ func TestPrepullImagesInParallel(t *testing.T) {
 		{
 			name:        "should error out; create failed",
 			p:           NewFailedCreatePrepuller(),
-			timeout:     10 * time.Second,
+			timeout:     PrepullImagesInParallelTimeout,
```
Comment (yagonobre) [inline cmd/kubeadm/app/phases/upgrade/prepull_test.go]: You should use `constants.PrepullImagesInParallelTimeout`

Comment (SataQiu) [reply cmd/kubeadm/app/phases/upgrade/prepull_test.go]: thanks 🤣

**Review by neolit123 — COMMENTED** [review]
thanks
/approve

**Pull request merged**